src/synapse/dataProvider.js

```diff
@@ -102,6 +102,16 @@ const resourceMap = {
     }),
     data: "members",
   },
+  pushers: {
+    map: p => ({
+      ...p,
+    }),
+    reference: id => ({
+      endpoint: `/_synapse/admin/v1/users/${id}/pushers`,
+    }),
+    data: "pushers",
+    total: json => json.total,
```

Comment:
needs to be a function for getManyReferences.. e.g. `total: json => {
      return json.total;
    },`